src/test/lib/storage/index/partial_hash/partial_hash_index_test.cpp

```diff
@@ -164,8 +164,10 @@ TEST_F(PartialHashIndexTest, Iterators) {
   EXPECT_EQ(std::distance(begin, end), 14);
   // Test size of NULL values index iterator.
   EXPECT_EQ(std::distance(null_cbegin(index), null_cend(index)), 2);
-  // Test for not-existing value in iterator.
+  // Test for existing value in iterator.
   EXPECT_NE(std::find(begin, end, RowID{ChunkID{0}, ChunkOffset{4}}), end);
+  // Test for not-existing value in iterator.
+  EXPECT_EQ(std::find(begin, end, RowID{ChunkID{2}, ChunkOffset{4}}), end);
 }
 
 TEST_F(PartialHashIndexTest, AccessWithIterators) {
```

Comment:
There are multiple test cases testing different functions of the partial index with different functors as parameters. Do you see any benefit in having them in one large `TEST_F` definition? I would argue that the tests are more structured, less complex and easier to understand if we have one `TEST_F` definition per actual test case.

Reply:
Good idea!